main_loop.py

```diff
@@ -1,38 +1,65 @@
 import pygame
+import os
+
 from room import Room
 from thing import Thing
 from monster import Monster
 
 
-pygame.init()
+class Game:
+    def __init__(self):
+        self.over = False
```

Comment:
Toto premenujte asi na `game_over` z `over` sa fakt neda pochopit co to je (a `game.over` vobec nie je vyhra...)